src/main/java/org/web3j/sample/Application.java

```diff
@@ -46,6 +47,7 @@
 public class Application {
 
     private static final Logger log = LoggerFactory.getLogger(Application.class);
+    Scanner s = new Scanner(System.in);
```

Comment:
Don't worry about this stuff, I'd rather that we leave out the scanner and don't request details from users, leaving placeholders in the code as was previously the case.

Reply:
Ok, I updated this now. Sorry for the confusion!

Reply:
One last change - you need to remove references to the Scanner on this line and in the import.